README.md

```diff
@@ -1,4 +1,7 @@
 # Diagnostics Working Group
+
+[![Join us on Slack](https://img.shields.io/badge/Join%20us%20on-Slack-e01563.svg)](https://node-js.slack.com/archives/C016U31J15M)
```

Comment:
```suggestion
[![Join us on Slack(#diagnostics-wg)](https://img.shields.io/badge/Join%20us%20on-Slack-e01563.svg)](https://node-js.slack.com/archives/C016U31J15M)
```

Reply:
The name of the channel is useful if you are already a member, suggest we add it.